proto/data_objects.proto

```diff
@@ -91,3 +85,51 @@ message DataObject {
   // A set of key-value pairs that represent metadata provided by the uploader.
   map<string, string> user_metadata = 13;
 }
+
+// Data Bundle: A collection of Data Objects
+message DataBundle {
+  // REQUIRED
+  // An identifier, unique to this Data Bundle
+  string id = 1;
+
+  // REQUIRED
+  // The list of Data Objects that this Data Bundle contains.
+  repeated DataObject data_objects = 2;
+
+  // REQUIRED
+  // Timestamp of object creation in RFC3339.
+  google.protobuf.Timestamp created = 3;
+
+  // REQUIRED
+  // Timestamp of update in RFC3339, identical to create timestamp in systems
+  // that do not support updates.
+  google.protobuf.Timestamp updated = 4;
+
+  // REQUIRED
+  // A string representing a version, some systems may use checksum, a RFC3339
+  // timestamp, or incrementing version number. For systems that do not support
+  // versioning please use your update timestamp as your version.
+  string version = 5;
+
+  // REQUIRED
+  // At least one checksum must be provided.
+  // The data bundle SHA256 checksum is computed over all the
+  // SHA256 checksums of the Data Objects that make up this bundle.
+  repeated Checksum checksum =  6;
```

Comment:
Can this be computed by iterating over the dataobject records only?  Or, does one need to download and compute over the actual files?

Reply:
For the case of Data Bundles I believe the intention was to only concatenate the checksums of the underlying data objects.

Reply:
Change the default to MD5. Specify more clearly how to concatenate MD5 digests.